src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Comment } from '../../types/Comment';
+import './CommentInfo.scss';
+
+type CommentInfoProps = {
```

Comment:
Це типово називати цей тип просто Props. 
```suggestion
type Props = {
```